sceptre/cli.py

```diff
@@ -235,6 +235,22 @@ def describe_stack_resources(ctx, environment, stack):
     write(response, ctx.obj["output_format"])
 
 
+@cli.command(name="diff-stack")
+@stack_options
+@click.pass_context
+@catch_exceptions
+def diff_stack(ctx, environment, stack):
+    """
+    Shows the local template differences against cloudformation.
+
+    Prints differences.
+    """
+    env = get_env(ctx.obj["sceptre_dir"], environment, ctx.obj["options"])
+    response = env.stacks[stack].diff()
+
+    write(response)
```

Comment:
```response``` should either be called ```diff``` or 

``` write(response)```
should be rewritten to be
```write(env.stacks[stack].diff())```
I have a weak preference for renaming "response" to "diff".